spec/controllers/items_controller_spec.rb

```diff
@@ -17,7 +17,7 @@
   describe '#create' do
     it 'should save an instance of Item to the db' do
       params = { item: { name: 'Table', length: 30, width: 60 } }
-      post :create, params: params
+      expect { post :create, params: params }.to change{ Item.count }.by(1)
```

Comment:
Rachael, changed your code here a bit to test this syntax, hope it's okay!

Reply:
No worries! Thank you!